resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
     END
 
 User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To DBaaSPolicy Screen    ${namespace}
-    Click If Element Present    ${lblOperand}
-    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
-    Click Element    ${btnCreateDBaaSPolicy}
-    Create Custom DBaaSPolicy as ${DBaaSPersona}
+    User Creates DBaaSPolicy For ${DBaaSPersona}
 
 User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Dev View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
```

Comment:
What is this ${operatorNamespace}?

Reply:
${operatorNamespace} Already in use and I am reusing this variable here